coderedcms/templates/coderedcms/blocks/accordion_block.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'coderedcms/blocks/base_block.html' %}
+
+{% load wagtailcore_tags coderedcms_tags %}
+
+{% block block_render %}
+
+{% generate_random_id as modal_id %}
```

Comment:
Since we have a snippet here, we can use the snippet's ID rather than having to generate a random one. The generation is necessary for arbitrary blocks in the streamfield that don't have IDs. Something like `snippet-{id}` would work fine here.